core/cap-0038.md

```diff
@@ -585,7 +596,18 @@ index 75f39eb4..505edbe5 100644
  };
  
  union RevokeSponsorshipResult switch (RevokeSponsorshipResultCode code)
-@@ -1229,6 +1354,63 @@ default:
+@@ -1218,7 +1345,9 @@ enum SetTrustLineFlagsResultCode
+     SET_TRUST_LINE_FLAGS_MALFORMED = -1,
+     SET_TRUST_LINE_FLAGS_NO_TRUST_LINE = -2,
+     SET_TRUST_LINE_FLAGS_CANT_REVOKE = -3,
+-    SET_TRUST_LINE_FLAGS_INVALID_STATE = -4
++    SET_TRUST_LINE_FLAGS_INVALID_STATE = -4,
++    SET_TRUST_LINE_FLAGS_REVOKE_LOW_RESERVE = -5 // claimable balances can't be created
```

Comment:
Should we just go with `SET_TRUST_LINE_FLAGS_LOW_RESERVE` (and the same for `ALLOW_TRUST`)? That's the model we typically use, and
- I'm not sure if the `REVOKE` adds any clarity
- What if we later have other reasons for low reserve?

Reply:
I was thinking the `REVOKE ` would be helpful since this is a very specific scenario, but I think potential future reasons for low reserve is a good enough reason to change this.